src/attribute.rs

```diff
@@ -0,0 +1,119 @@
+use crate::configuration::Path;
+use crate::filter::http_context::Filter;
+use chrono::{DateTime, FixedOffset};
+use proxy_wasm::traits::Context;
+
+pub trait Attribute {
+    fn parse(raw_attribute: Vec<u8>) -> Result<Self, String>
+    where
+        Self: Sized;
+}
+
+impl Attribute for String {
```

Comment:
I think this could "just" be the `TryInto` trait, but it is that against the [orphan rules](https://doc.rust-lang.org/reference/items/implementations.html?highlight=orphan#orphan-rules)